cmd/importcorpora/main.go

```diff
@@ -0,0 +1,203 @@
+// Usage: go run cmd/importcorpora/main.go
+//
+// Updates the at the bottom of this file specified data files with content from dariusk/corpora.
+package main
+
+import (
+	"archive/zip"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// URL to zip file
+const corporaArchive = "https://github.com/dariusk/corpora/archive/master.zip"
+
+// Path to write Go file to
+const targetDir = "pkg/data"
+
+// Content of a Go file
+const fileTemplate = `package data
+
+var %s = %s`
+
+func main() {
+	// Check if running in repo directory
+	_, err := os.Stat(targetDir)
+	if err != nil && !os.IsNotExist(err) {
+		log.Fatal(err)
+	}
+	if err != nil {
+		log.Fatalf("The data directory cannot be found at %s. Ensure the importer is running in the correct location.", targetDir)
+	}
+
+	dir, err := downloadCorpora()
+	defer func() {
+		if dir == "" {
+			return
+		}
+		if err := os.RemoveAll(dir); err != nil {
+			panic(err)
+		}
+	}()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, d := range data {
+		// Get data from JSON file
+		f, err := ioutil.ReadFile(filepath.Join(dir, "corpora-master/data", d.From))
+		if err != nil {
+			log.Fatal(err)
+		}
+		var jsonData map[string]interface{}
+		if err := json.Unmarshal(f, &jsonData); err != nil {
+			log.Fatal(err)
+		}
+
+		// Fix formatting
+		value := strings.Replace(fmt.Sprintf("%#v\n", jsonData[d.Key]), "interface {}", "string", 1)
```

Comment:
If anyone has a nice way of directly getting s string slice from the JSON, would be happy to change this :)